src/EditorFeatures/VisualBasicTest/AutomaticEndConstructCorrection/XmlElementTagCorrectorTests.vb

```diff
@@ -0,0 +1,39 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Utilities
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.AutomaticEndConstructCorrection
+Imports Microsoft.VisualStudio.Text
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.AutomaticEndConstructCorrection
+    Public Class XmlElementTagCorrectorTests
+        Inherits AbstractCorrectorTests
+
+        Friend Overrides Function CreateCorrector(buffer As ITextBuffer, waitIndicator As TestWaitIndicator) As ICorrector
+            Return New XmlElementTagCorrector(buffer, waitIndicator)
+        End Function
+
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.AutomaticEndConstructCorrection)>
+        Public Sub TestXmlStartTag()
+            Dim code = <code><![CDATA[
+''' <[|summary|]></[|summary|]>
+Structure A
+End Structure
+]]></code>.Value
+
+            VerifyBegin(code, "see")
+        End Sub
+
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.AutomaticEndConstructCorrection)>
+        Public Sub TestContinuousEdit2()
+            Dim code = <code><![CDATA[
+''' <[|$$summary|]></[|summary|]>
+Structure A
+End Structure
+]]></code>.Value
+
+            VerifyContinuousEdits(code, "see", Function(s) s, removeOriginalContent:=True)
+        End Sub
+    End Class
+End Namespace
```

Comment:
4. Why? By the looks of the code, i think it will work in xml literals without requiring more changes. Is it not the case? I still think adding it to xml literals is good.
6. If the xml is going to be well formed, the user has to insert matching tags for the tags that they just "broke". correct? so i thought by updating the start tag the user has less typing to do, even if they are inserting new child tags. I can understand the reason to not do it if that makes it looks very aggressive